packages/cxl-lumo-styles/scss/cxl-vaadin-button.scss

```diff
@@ -0,0 +1,106 @@
+.vaadin-button {
```

Comment:
Add comment with a GH release tag source tree for where these styles came from.
Many examples through codebase available.